### PR TITLE
Add logo to the docs, and use chemiscope for trajectory visualization

### PR DESCRIPTION
**Pull request by Luthaf**
Some small improvements to the documentation. The chemiscope changes need https://github.com/lab-cosmo/chemiscope/pull/345, so we'll have to wait until the PR over there is merged and released on PyPI.

# Contributor (creator of pull-request) checklist

 - [ ] Tests updated (for new features and bugfixes)?
 - [ ] Documentation updated (for new features)?
 - [ ] Issue referenced (for PRs that solve an issue)?

# Reviewer checklist

 - [ ] CHANGELOG updated with public API or any other important changes?

----
 📚 [Download documentation preview for this pull-request](https://nightly.link/lab-cosmo/metatensor/actions/artifacts/1608043116.zip)

### CODE REVIEW

File: python/examples/atomistic/2-running-ase-md.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 .. TODO: once we have more MD engine, add link to their doc here
 """
 
-# sphinx_gallery_thumbnail_number = 4
```
Comment (PicoCentauri) [inline python/examples/atomistic/2-running-ase-md.py:16]: why are you changing this if it is commented out?

Comment (Luthaf) [reply python/examples/atomistic/2-running-ase-md.py:16]: this is the syntax sphinx-gallery uses to have configuration inside the examples: https://sphinx-gallery.github.io/stable/configuration.html#configurations-inside-examples

Comment (PicoCentauri) [reply python/examples/atomistic/2-running-ase-md.py:16]: Ahh true I remember now

**Review by PicoCentauri — APPROVED** [review]
Wonderful!

**Comment by Luthaf** [comment]
Thanks!

**Pull request merged**